docker-compose.yml

```diff
@@ -10,24 +10,39 @@ services:
     volumes:
       - database:/var/lib/postgresql/data
 
-  app:
+  redis:
+    env_file: .env
+    expose:
+      - 6379
+    image: redis:5.0.6-alpine
+    tty: true
+
+  common: &common
     build:
       context: .
-      target: app
-    command: gunicorn developerportal.wsgi:application --bind=0.0.0.0:8000 --reload --workers=3
+      target: app_base
     depends_on:
       - db
+      - redis
     env_file: .env
-    ports:
-      - "8000:8000"
-      - "8080:8080"
     tty: true
     volumes:
       - dist:/app/dist
       - ./developerportal:/app/developerportal
       - ./media:/app/media
       - ./src:/app/src
 
+  app:
+    <<: *common
+    command: gunicorn developerportal.wsgi:application --bind=0.0.0.0:8000 --reload --workers=3
+    ports:
+      - '8000:8000'
+      - '8080:8080'
+
+  worker:
+    <<: *common
+    command: celery -A developerportal.apps.staticbuild worker -l info
```

Comment:
The only niggle I can see with running celery like this is the warning

`RuntimeWarning: You're running the worker with superuser privileges: this is absolutely not recommended!`

@escattone / @limed Do either of you have a suggestion of how to tackle this, please, or if we def need to (given we don't use this docker-compose.yml in k8s, yep?)

Reply:
Docker recommend [running services as a non-privileged user][1] if possible. It sounds like that would fix the warning but I’m not sure what else it would impact.

[1]: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user